### DISCUSSION TITLE
SSL support?

### DISCUSSION THREAD

**Discussion by bsara** [afs/rdf-delta#63] [Q&A] [answered]
I'm having trouble finding any documentation on how to enable SSL for the delta patch server. Is this available? If so, how is configured?

**Comment by afs** [comment, marked as answer]
The patch server has an argument:

`    --jetty=FILE        File name of a jetty.xml configuration file.`

so an HTTPS setup can be used by configuration that way. 

It's a bit raw - there is helper code in Fuseki (class `JettyHttps`) but that hasn't made its way to the RDF delta codebase yet.

---

One configuration setup I have seen, is to put the patch serves on their own VPN, and have controls on the VPN to only talk to the VPN of the Fuseki servers.

**Reply by bsara** [reply]
thanks!
